import_export/views.py

```diff
@@ -0,0 +1,88 @@
+from django.http import HttpResponse
```

Comment:
Wouldn't it be better to rename module to `mixins.py`?

Reply:
I changed to `mixins.py`